src/draftjsToMd.js

```diff
@@ -76,21 +76,24 @@ function fixWhitespacesInsideStyle(text, style) {
   // Text between opening and closing markers
   const body = text.slice(style.range.start, style.range.end);
   // Trimmed text between markers
-  const bodyT = body.trim();
+  const bodyTrimmed = body.trim();
   // Text after closing marker
   const post = text.slice(style.range.end);
 
+  const bodyTrimmedStart = style.range.start + body.indexOf(bodyTrimmed);
+
   // Text between opening marker and trimmed content (leading spaces)
-  const prefix = text.slice(style.range.start, text.indexOf(bodyT));
```

Comment:
The bug was on this line, it would match outside the trimmed content so that it finds the first "is" in:

`this __is__ a test`

rather than the one being styled